enhancements/update/cluster-profiles.md

```diff
@@ -105,6 +92,55 @@ the required manifests while keeping a single master source.
 The current installation profile is called `self-managed-high-availability`. All current
 manifests must specify it. Future profiles may choose.
 
+#### Cluster Profile for Cluster Version Operator design
+
+A cluster profile is specified to the CVO as an identifier in, either:
+ * an environment variable,
+ * the `status.ClusterProfile` properties in `ClusterVersion`.
```

Comment:
I don’t see a reason to do this because it exposes the wrong behavior for clients to trigger on.  What are the use cases that you would read this value (ie, why are you trying to do this)?  What characteristics of the profile are you trying to react to?

Reply:
This solution was a reaction of the push back from the installer team. See https://github.com/openshift/installer/pull/3986#issuecomment-666494951

I don't want to react on this profile. This was just a mean to pass the cluster profile without having to change the installer.

```
openshift-install create manifests
patch the CV
openshift-install create cluster
```
 
For sure, I will be happier if I can pass it directly in the install-config.yaml.

An argument for this new property is that it can be used in insights and telemetry.

Reply:
is this focused on the CRC case at the moment?  it is disabling insights and telemetry by default....

Reply:
while we do phase 0 of the crc related enhancement, is the patch step enumerated above sufficient to make improved incremental progress without committing to an externally user facing API?  it seems like a reasonable first-step to me.

Reply:
We will need to use cluster profile for https://github.com/openshift/enhancements/pull/504 in 4.8, so it's not only CRC oriented